psalm-baseline.xml

```diff
@@ -487,15 +487,6 @@
       <code>matching</code>
     </UndefinedInterfaceMethod>
   </file>
-  <file src="lib/Doctrine/ORM/Mapping/Builder/ClassMetadataBuilder.php">
-    <ArgumentTypeCoercion occurrences="2">
-      <code>$class</code>
-      <code>$repositoryClassName</code>
-    </ArgumentTypeCoercion>
```

Comment:
You should restore this

Reply:
Ok, we still need this rule `ArgumentTypeCoercion` 👍🏾